wasm/src/object.c

```diff
@@ -1,5 +1,76 @@
 #include "object.h"
 
+
+static opa_value *__merge(opa_value *a, opa_value *b);
+static opa_value *__merge_with_overwrite(opa_value *a, opa_value *b);
+static void __copy_object_elem(opa_object_t *result, opa_value *a, opa_value *b);
```

Comment:
💭 So this is how you declare functions that aren't callable outside of this .c file? I.e., when another file imports `object.h`, it wouldn't learn about these. Do you _need_ these declarations? Or is it the `static` that we do this for?

Reply:
We need to pre-declare the helpers but they don't have to be static. Since they are helpers I declared them as static. Let me know if you think we should update the declarations.